pkg/utils/k8sutil.go

```diff
@@ -151,6 +151,20 @@ func GetHTTPTriggerClientOutCluster() (versioned.Interface, error) {
 	return kubelessClient, nil
 }
 
+// GetKubelessClientOutCluster returns kubeless clientset for making requests from outside of cluster
+func GetKubelessClientOutCluster() (versioned.Interface, error) {
```

Comment:
there is no difference between this function and the other `Get*ClientOutCluster`, why don't we merge them?